scripts/xsslint/xsslint/main.py

```diff
@@ -0,0 +1,140 @@
+"""
+The main function for the XSS linter.
+"""
+import argparse
+import os
+import sys
+
+from xsslint.linters import JavaScriptLinter, MakoTemplateLinter, PythonLinter, UnderscoreTemplateLinter
```

Comment:
Does it work to add relative imports on all files in this xsslint subdirectory.  For example:
```
from .linters import JavaScriptLinter, MakoTemplateLinter, PythonLinter, UnderscoreTemplateLinter
```

Reply:
What would be the advantage in doing so? We don't seem to use relative imports very much throughout the codebase.

Reply:
We don't have an OEP for it, but if you search in Confluence, you can see various pages that mention this best practice:
https://openedx.atlassian.net/wiki/spaces/EDUCATOR/pages/251528559/Coding+Practices

Yes.  There is code that doesn't follow this practice, but it is best practice. :)

FYI: @nasthagiri - I can add it to my list, but it will take a while. :)

Reply:
We follow [PEP8](https://www.python.org/dev/peps/pep-0008/#imports), which says:

“Absolute imports are recommended, as they are usually more readable and tend to be better behaved.. However, explicit relative imports are an acceptable alternative to absolute imports, especially when dealing with complex package layouts where using absolute imports would be unnecessarily verbose.”

So I wouldn’t say that always using relative imports is edX “best practice”.   Rather, I’ve found that absolute imports have been very useful for finding usages and refactoring.

Reply:
@amangano-edx: To clarify, I already approved and you should feel free to squash and merge.

Thanks for your comment @nasthagiri.  It was a surprise to me.  You noted the advantage of absolute imports, but there are also advantages of relative imports.  Also, I learned what I thought was the best practice from other developers, so I think it would be great for us to clarify this one outside of PEP8 to get agreement and have an edX OEP to point to.